rhel8/profiles/cis.profile

```diff
@@ -710,52 +710,67 @@ selections:
     ## 5.2 SSH Server Configuration
 
     ### 5.2.1 Ensure permissions on /etc/ssh/sshd_config are configured (Scored)
+    # chown root:root /etc/ssh/sshd_config
+    - file_owner_sshd_config
+    - file_groupowner_sshd_config
 
+    # chmod og-rwx /etc/ssh/sshd_config
+    - file_permissions_sshd_config
 
     ### 5.2.2 Ensure SSH access is limited (Scored) 
 
 
     ### 5.2.3 Ensure permissions on SSH private host key files are
     ###       configured (Scored)
+    # TO DO: The rule sets to 640, but benchmark wants 600
+    - file_permissions_sshd_private_key
+    # TO DO: check owner of private keys in /etc/ssh is root:root
 
-
-    ### 5.2. Ensure permissions on SSH public host key files are configured
+    ### 5.2.4 Ensure permissions on SSH public host key files are configured
     ###      (Scored)
-
+    - file_permissions_sshd_pub_key
+    # TO DO: check owner of pub keys in /etc/ssh is root:root
 
     ### 5.2.5 Ensure SSH LogLevel is appropriate (Scored)
-
+    - sshd_set_loglevel_info
 
     ### 5.2.6 Ensure SSH X11 forward is diabled (Scored)
-
+    - sshd_enable_x11_forwarding
```

Comment:
We want the forwarding disabled, not enabled

Reply:
Nice catch, thank you. Removed the rule.